src/libraries/System.IO.MemoryMappedFiles/ref/System.IO.MemoryMappedFiles.csproj

```diff
@@ -12,5 +12,6 @@
     <ProjectReference Include="..\..\System.IO.UnmanagedMemoryStream\ref\System.IO.UnmanagedMemoryStream.csproj" />
     <ProjectReference Include="..\..\System.Runtime.InteropServices\ref\System.Runtime.InteropServices.csproj" />
     <ProjectReference Include="..\..\System.Runtime\ref\System.Runtime.csproj" />
+    <ProjectReference Include="..\..\System.Security.AccessControl\ref\System.Security.AccessControl.csproj" />
```

Comment:
So now you have `System.IO.MemoryMappedFiles` (part of the netcoreapp refs, does not ship as package) depending on `System.Security.AccessControl` (still ships as a package).  That does not look right.

E.g. I believe that this means that `System.Security.AccessControl` ref needs to be part of what you get by default for netcoreapp. But `System.Security.AccessControl` is Windows-specific and the whole point of this exercise was to not fault it into the platform-agnostic netcoreapp by default.

Reply:
I don't think this is needed anymore. This seems like a leftover from when we started figuring out the structure of this.

Reply:
Removed. I built locally and it succeeded.